test/unit/typography/font_loading.js

```diff
@@ -59,7 +59,7 @@ suite('Fonts outside preload()', function () {
 
     test('should call success-callback with object when font loads',
       function (done) {
-        p5Inst.loadFont('manual-test-examples/p5.Font/acmesa.ttf',
+        p5Inst.loadFont('../manual-test-examples/p5.Font/acmesa.ttf',
```

Comment:
I feel a little bit weird about all these links that go up one level `../`. I wonder if there is a way to set a working directory to run the tests from that is the `test/` folder? if this can't be done easily, this isn't a big point, just wanted to ask.